src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -21,57 +21,24 @@ public class HelpWindow extends UiPart<Stage> {
             + "   H E L P   M E N U\n"
             + "============================================================================================\n"
             + "\n1. Add Student\n"
-            + "   - Purpose: Adds a student and their details to the address book.\n"
-            + "   - Command Format:\n"
-            + "       add id/ [STUDENT_ID] n/ [STUDENT_NAME] p/ [PHONE_NUMBER] a/ [ADDRESS] c/ [COURSE] t/ [TAGS]\n"
-            + "   - Example:\n"
-            + "       add id/ 12345678 n/ John Doe p/ 99999999 a/ 123 Jane Doe Road "
-            + "c/ Computer Science t/ Student\n"
+            + "   - add id/ [ID] n/ [STUDENT_NAME] p/ [PHONE_NUMBER] a/ [ADDRESS] c/ [COURSE] t/ [TAGS]\n"
```

Comment:
I think the format should be 
```suggestion
            + "   - add [ID] n/ [STUDENT_NAME] p/ [PHONE_NUMBER] a/ [ADDRESS] c/ [COURSE] t/ [TAGS]\n"
```
since we are removing id prefix for all commands?